stl/inc/xutility

```diff
@@ -3246,11 +3246,14 @@ _NODISCARD constexpr const tuple_element_t<_Index, tuple<_Types...>>&& get(const
 
 namespace ranges {
     // clang-format off
+    template<class _From, class _To>
+    concept _Uses_nonqualification_pointer_conversion =
+      is_pointer_v<_From> && is_pointer_v<_To> &&
+      !convertible_to<remove_pointer_t<_From>(*)[], remove_pointer_t<_To>(*)[]>;

```

Comment:
Style nitpicks:
* There should be a space after `template`
* Indentation should be 4 spaces
* Operators should appear at the beginning of the line (this makes them more visible)